main.go

```diff
@@ -61,6 +61,7 @@ func main() {
 		cli.WithCLIVersion(buildVersion, buildTime),
 		cli.WithKubernetesNodeVersion(k8sVersion),
 		cli.WithProvider("systemd", func(cfg provider.InitConfig) (provider.Provider, error) {
+			cfg.ConfigPath = o.KubeConfigPath
```

Comment:
Just to be sure `o.KubeConfigPath` is populated by flags, you don't have env var `KUBECONFIG` set, do you?

Reply:
indeed, that env vars seems completely separate from the value of the --kubeconfig flag. I rather use 1 entry point to get the config.